docs/theming.md

```diff
@@ -430,37 +430,25 @@ Since our cards only contain text and no components, let's keep it simple for no
 > Note: in the future we plan to provide a Javascript utility method for changing all derived colors and making this
 use-case easier.
 
+## Custom Sass Mixins
 
-## Dark Themes
+For example, let's say you want to make particular
```

Comment:
Let's rephrase this paragraph to something like:

```
Most MDC Web components provide a set of Sass mixins to customize their appearance,
such as changing the fill color, ink color, stroke width, etc.
These mixins are documented in each component's README file
(e.g., the [Button readme](../packages/mdc-button/README.md#advanced-sass-mixins)).
```

And line wrap at 120 chars 🙂